content/docs/5.1/concepts/explore-tikv-features/fault-tolerance.md

```diff
@@ -45,40 +51,41 @@ To view the Grafana: http://127.0.0.1:3000
 ```
 
 {{< info >}}
-Each region contains 3 replicas according to the default configuration.
+Each Region has three replicas according to the default configuration.
 {{< /info >}}
 
-## Step 2. Write data
+### Step 2. Import data to TiKV
 
-On another terminal session, use [go-ycsb](https://github.com/pingcap/go-ycsb) to launch a workload.
+On another terminal session, use [go-ycsb](https://github.com/pingcap/go-ycsb) to launch a workload of writing data to the TiKV cluster.
 
-1. Clone the `go-ycsb` from GitHub.
+1. Clone `go-ycsb` from GitHub.
 
-    ```sh
+    ```shell
```

Comment:
We may use `sh` instead, `shell` will cancel the code highlight of some editors like vscode.

Reply:
```suggestion
    ```sh
```